### PR TITLE
Matcher Update

### PR DESCRIPTION
**Pull request by acpaquette**
Changed ring matcher to have the same columns as the k nearest neighbors matcher for the source_image and destination_image

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,20 +32,6 @@ def setup_package():
...
-              'vlfeat',
-              'cyvlfeat',
-              'csmapi',
-              'networkx',
-              'pillow',
-              'plio',
-              'ncurses',
-              'geopandas',
-              'imageio'],
         classifiers=[
```
Comment (jlaura) [inline setup.py]: Why this change? Is it because everything is in the environment.yml?

File: autocnet/graph/edge.py [hunk 2]
```diff
@@ -185,11 +185,11 @@ def ring_match(self, *args, **kwargs):
...
         matches[:,3] = tar_kps.index[pidx[:,1]].values
 
-        matches = pd.DataFrame(matches, columns=['source',
+        matches = pd.DataFrame(matches, columns=['source_image',
                                                  'source_idx',
-                                                 'destination',
+                                                 'destination_image',
                                                  'destination_idx']).astype(np.float32)
-        
+
```
Comment (jlaura) [inline autocnet/graph/edge.py]: :+1:

File: setup.py (hunk 1 above)
Comment (Kelvinrr) [reply setup.py]: If I recall correctly, he was trying to run `setup.py develop` and it couldn't find one of the packages required despite it being conda installed, so I just told him to blast the list as a quick fix. Since things are all in the env file I don't see why they can't stay gone.

Comment (jlaura) [reply setup.py]: :+1:
